proto/osmosis/twap/v1beta1/query.proto

```diff
@@ -16,7 +16,12 @@ option go_package = "github.com/osmosis-labs/osmosis/v11/x/twap/client/queryprot
 service Query {
   rpc GetArithmeticTwap(GetArithmeticTwapRequest)
       returns (GetArithmeticTwapResponse) {
-    option (google.api.http).get = "/osmosis/gamm/v1beta1/pools";
+    option (google.api.http).get = "/osmosis/twap/v1beta1/GetArithmeticTwap";
+  }
+  rpc GetArithmeticTwapToNow(GetArithmeticTwapToNowRequest)
```

Comment:
Sorry late to this PR, but since we're using a template to generate grpc queries, i think it'd be better if we have a naming standard for requests and responses. 

For instance, Gamm queries start with "query" like `QueryPoolsRequest` `QueryTotalLiquidityRequest` `QuerySpotPriceResponse` 

However, Twap queries doesnot contain "query" prefix like `GetArithmeticTwapToNowRequest`